.devcontainer/cpu/devcontainer.json

```diff
@@ -17,7 +17,7 @@
             ]
         }
     },
-
+    "runArgs": ["--shm-size=4gb"],
     "overrideCommand": true,
-    "postStartCommand": "pip install -e ."
+    "postStartCommand": "pip install -e .[dev]"
```

Comment:
Revert these changes. They're outside the scope of your issue and they are handled by us.